website/docs/dev_launcher.md

```diff
@@ -90,6 +90,10 @@ Environment variables that are set during startup:
 - **AYON_HEADLESS_MODE** - Headless mode flag enabled when set to '1'.
 - **AYON_EXECUTABLE** - Path to executable that is used to run AYON.
 - **AYON_ROOT** - Root to AYON launcher content.
+- **AYON_LAUNCHER_STORAGE_DIR** - Directory where are stored dependency packages, addons and files related to addons.
+- **AYON_LAUNCHER_LOCAL_DIR** - Directory where are stored user/machine specific files. This MUST NOT be shared.
```

Comment:
I can't get the difference between these two.

Reply:
What exactly you don't understand? `STORAGE_DIR` is for files that can be shared, and `LOCAL_DIR` is for machine specific files dir should not be shared.

Reply:
`AYON_LAUNCHER_STORAGE_DIR` which is the Storage directory that is used for storing shims, addons, dependencies, etc.
`AYON_LAUNCHER_LOCAL_DIR` still doesn't seem clear to me. only example I found was for using it for temp data like thumbnails cache.

Reply:
For example file with site id, tools are storing states of the tools there (e.g. python console interpreter), some addons are storing their caches, none of it should be shared.

Reply:
Cool, Could we elaborate on the docs to reflect these differences?
Maybe, 
```suggestion
- **AYON_LAUNCHER_STORAGE_DIR** - Directory where the launcher saves downloaded dependency packages, addons, and files associated with those add-ons.
- **AYON_LAUNCHER_LOCAL_DIR** - Directory where the launcher stores temporary cache files that are specific to the user or machine, such as tool states and thumbnails. This MUST NOT be shared between different machines.
```

Reply:
I don't think it is necessary. I would not mention launcher there at all, it is not launcher who does all the things. Any addon can add there those files, you can add those files on your own. Too much information is more confusing in this case. For example `"and files associated with those add-on"` can be said for both env variables. It depends on the usage of those files if are added to one or other. I would not mention addon specific information at all (like thumbnails) as that is not defined by launcher, that is defined by ayon-core, which can change that logic at any point.

Reply:
I hope if we could find a better suggestion. because honestly, I find the current definitions confusing when reading it for the first time :/  
and, no way I was able to understand what are these without starting a discussion about them and then go searching the code base for answers to see when each variable is used.

Reply:
I agree with Kuba - it's clear already. We don't know yet exactly what will end up in both folders over time so let's leave that open.

Reply:
Nevermind. let's leave for the future.

Reply:
Yeah, let's leave for the future